package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "react-spreadsheet",
-  "version": "0.8.4",
+  "name": "@silicon-jungle/fork-react-spreadsheet",
```

Comment:
This is probably a mistake. Must be "react-spreadsheet".

Reply:
Yeah, this should not be included. I was just running this as a test. Not sure why I pushed this up. Will fix.